ca.desrt.dconf-editor.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "ca.desrt.dconf-editor",
+    "base-version": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "dconf-editor",
+    "rename-icon": "dconf-editor",
+    "copy-icon": true,
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--socket=wayland",
+	    "--talk-name=ca.desrt.dconf"
```

Comment:
That is actually true! I'd gone back and forth about where to grant - this is what comes of doing my testing on a very base OS. Thanks